admin-ui/plugins/admin/components/Webhook/WebhookURLChecker.js

```diff
@@ -0,0 +1,19 @@
+const NOT_ALLOWED = ["http://", "ftp://", "file://", "telnet://", "smb://", "ssh://", "ldap://", "https://192.168", "https://127.0", "https://172", "https://localhost"]
+export const isValid = (url) => {
+    if (url === undefined || url === null) {
+        return false;
+    } else {
+        return isAllowed(url)
+    }
+}
+
+const isAllowed = (url) => {
```

Comment:
I think if we add `https://local` as an input the `isValid` will return true even if the url is invalid. Could you check [this answer](https://stackoverflow.com/a/61691586/13718296) on stackoverflow. For validation they are using Yup as we do for other fields.

Reply:
@jv18creator the validation we need on the Ui is just to block unwanted urls, we don't want to check if the url is resolvable or no. 

That check will be done on the backend when the webhook is enable and about to be schedule.

Reply:
But i have added the Regex. Regading Yup in this case i don't think we need it as we have complexe validation, regex and blacklist).